### PR TITLE
Fea/debug seg

### PR DESCRIPTION
**Pull request by jxchen01**
update installation instruction

improve debugging mode for semantic segmentation problem by applying softmax before saving predictions

### CODE REVIEW

**Review by funkyZhaofour — COMMENTED** [review]
1. Tell us how to install the right vision of pytorch, torchvision and CUDA environment.
2.and 3. Tell us that if wan to do softmax about the output, we can set 'model_extra:
    debug_segmentation: True' in preset yaml.

**Review by Justin-Sonneck — APPROVED** [review]
